pom.xml

```diff
@@ -253,13 +253,10 @@
 				<repository>
 					<id>spring-snapshots</id>
 					<name>Spring Snapshots</name>
-					<url>https://repo.spring.io/libs-snapshot-local</url>
+					<url>https://repo.spring.io/snapshot</url>
 					<snapshots>
 						<enabled>true</enabled>
 					</snapshots>
-					<releases>
-						<enabled>false</enabled>
```

Comment:
I think this should still be set to `false` because we don't want to pull release versions from the snapshots repo.

Btw, the milestone repo below should also be updated to `https://repo.spring.io/milestone`.

Also, you should do the same for plugin repositories below.

Reply:
Done, changed it for all the URLs across the repo.